hudi-spark-datasource/hudi-spark/pom.xml

```diff
@@ -274,11 +278,23 @@
       <scope>test</scope>
     </dependency>
 
+    <dependency>
```

Comment:
Please add a comment elaborating on why we need this setup

Reply:
Added comment in latest commit.